src/backend/marsha/core/apps.py

```diff
@@ -3,9 +3,15 @@
 from django.apps import AppConfig
 from django.utils.translation import gettext_lazy as _
 
+from marsha.core import signals
+
 
 class CoreConfig(AppConfig):
     """Django app config for the ``core`` app."""
 
     name = "marsha.core"
     verbose_name = _("Marsha")
+
+    def ready(self):
+        """Register marsha handlers."""
+        signals.live_status_changed.connect(signals.handlers.delete_aws_element_stack)
```

Comment:
Semantically, I feel weird about connecting a method `delete_aws_element_pack` for all `update` events.